src/utils/theme.ts

```diff
@@ -0,0 +1,183 @@
+import { createTheme } from "@mui/material/styles";
+import { Color } from "./colors";
+
+const theme = createTheme({
+  palette: {
+    mode: "dark",
+    primary: {
+      main: Color.PRIMARY_DEFAULT,
+      light: Color.PRIMARY_HOVER,
+      dark: Color.PRIMARY_ACTION,
+    },
+    secondary: {
+      main: Color.SECONDARY,
+    },
+    info: {
+      main: Color.LINK,
+    },
+    success: {
+      main: Color.SUCCESS,
+    },
+    error: {
+      main: Color.ERROR,
+    },
+    warning: {
+      main: Color.WARNING_DEFAULT,
+      light: Color.WARNING_CRITICAL,
+      dark: Color.WARNING_HEAVY,
+    },
+    background: {
+      default: Color.PRIMARY_ACTION,
+    },
+  },
+  typography: {
+    allVariants: {
+      color: Color.NEUTRAL_WHITE,
+      wordWrap: "break-word",
+      wordBreak: "break-word",
+    },
+    fontFamily: '"Comme", sans-serif',
+    h1: {
+      fontFamily: '"Comme", sans-serif',
+      fontWeight: 700,
+      fontSize: "32px",
```

Comment:
In the global.css there are variables such as `--font-size-h1: 32px;`, it would be good if you put the same variable in your theme.ts file since it is currently like this:

```
h1: {
fontFamily: '"Comme", sans-serif',
fontWeight: 700,
fontSize: "32px",
},
```

and it would be good if you entered it so that it has more consistency in all styles.

Reply:
It is not possible to use a **CSS** variable in a **TS** (Typescript) file. I moved repeated typography values in the ts file for MUI theme data in constants like the colors.

Reply:
Alright